Tests/NIOTransportServicesTests/NIOTSBootstrapTests.swift

```diff
@@ -336,6 +336,29 @@ final class NIOTSBootstrapTests: XCTestCase {
             try? connectionChannel?.close().wait()
         }
     }
+
+    func testBootstrapsMultipath() throws {
+        let group = NIOTSEventLoopGroup()
+        defer {
+            try! group.syncShutdownGracefully()
+        }
+
+        let listenerBootstrap = NIOTSListenerBootstrap(group: group).withMultipath(.handover)
+        let connectionBootstrap = NIOTSConnectionBootstrap(group: group).withMultipath(.handover)
+
+        do{
+            var listenerChannel: Channel = try listenerBootstrap.bind(host: "localhost", port: 8000).wait()
```

Comment:
Let's not force port 8000: instead, let's bind port 0 and then use `localAddress` to get the target for the connection.

Reply:
It will be better indeed !